### PR TITLE
Update E2E Tests (Part 1)

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is various small fixes and improvements for E2E Tests

Part of #2003 

# Discussion

- Proxy pallet fixes
- Swap tests to use createSchemaV3
- Eslint globals to fix `npm run format`
- Fix missing version flag on runtime api to fix the missing runtime api notice

### CODE REVIEW

File: e2e/eslint.config.mjs
Comment (wilwade) [inline e2e/eslint.config.mjs:1]: Fixes `npm run lint` and format

File: e2e/package.json [hunk 1]
```diff
@@ -34,7 +34,9 @@
     "@types/workerpool": "^6.4.7",
     "eslint": "^8.57.0",
     "eslint-plugin-mocha": "^10.4.3",
+    "globals": "^15.3.0",
     "mocha": "^10.4.0",
+    "node-datachannel": "^0.9.1",
```
Comment (wilwade) [inline e2e/package.json:39]: Something in there is missing this dependency, so had to add it here.

File: e2e/proxy-pallet/proxy.test.ts [hunk 2]
```diff
@@ -93,7 +93,6 @@ describe('Proxy', function () {
       // Filtered Out ExtrinsicFailed
       await assert.rejects(extrinsic.signAndSend(), {
         name: 'NotProxy',
-        message: /Sender is not a proxy of the account to be proxied./,
```
Comment (wilwade) [inline e2e/proxy-pallet/proxy.test.ts:96]: These messages are not sent back when running against Testnet on Paseo

File: js/api-augment/definitions/frequencyTxPayment.ts [hunk 3]
```diff
@@ -35,6 +35,7 @@ export default {
             type: 'FeeDetails',
           },
         },
+        version: 1,
```
Comment (wilwade) [inline js/api-augment/definitions/frequencyTxPayment.ts:38]: Not sure why this wasn't here, but it fixes the error and aligns it with the other definitions

File: e2e/scaffolding/helpers.ts
Comment (wilwade) [inline e2e/scaffolding/helpers.ts:1]: `createSchemaV3` is the latest and greatest and these are not testing it, just using it elsewhere.

**Review by JoeCap08055 — APPROVED** [review]
One question, non-blocking, just for my own education. Otherwise lgtm.

File: e2e/proxy-pallet/proxy.test.ts (hunk 2 above)
Comment (wilwade) [reply e2e/proxy-pallet/proxy.test.ts:96]: It is the way the builds work. It has to do with how much is compiled into the runtime. I don't recall the specific details.

**Review by mattheworris — APPROVED** [review]
- Read through changes, all looks in order.
🚢 it!

File: e2e/scaffolding/helpers.ts
Comment (shannonwells) [reply e2e/scaffolding/helpers.ts:1]: Do we need to add some sort of ToDo for new schema versions to make sure this gets updated?

**Review by shannonwells — APPROVED** [review]
Non-blocking question, LGTM

File: e2e/scaffolding/helpers.ts
Comment (wilwade) [reply e2e/scaffolding/helpers.ts:1]: This is the helpers, so unless someone bypasses it everything should use it

**Pull request merged**